wilhelm-lab/dlomix
# Fix/gh actions and docs imporovements

## Changes:
### Docs:
- new doc theme 
- sphinx security dependency update
- refactoring of rst files
- addition of install, quick start in rst
- build and view docs locally with `make build-docs`

### GitHub Actions:
- fixes to allow pipeline steps run with correct dependency and upon success of required steps

### Tests:
- refactoring of tests and assets needed (tests run much faster)
- run tests locally with `make test-local`
- make file test runs ensure package is removed and re-installed

### pre-commits:
- added basic pre-commit hooks for formatting and sorting
